zypper-migration

```diff
@@ -106,6 +106,26 @@ def check_if_param(opt, message)
   end
 end
 
+def get_available_services_based_on_url
+  SUSE::Connect::Zypper::services.select { |service| service[:url].include?("plugin:/susecloud") || service[:url].include?("susecloud.net") }
```

Comment:
Same issue actually applies for BYOS, so I guess that should check for `suse.com` URLs (for services served by SCC).

Also I think that the URLs sometimes might be in `plugin:susecloud` format, so perhaps add it as well:
https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/c359df4059a28356585d5c37ed4a035d35b9ccf9/lib/cloudregister/registerutils.py#L665-L666